server/src/test/java/io/crate/planner/ViewPlannerTest.java

```diff
@@ -52,10 +52,10 @@ public void test_view_of_join_condition_containing_subscript_expressions() throw
             Rename[id] AS doc.v1
               └ Eval[id]
                 └ HashJoin[LEFT | (o['i'] = o['i'])]
-                  ├ Rename[o['i']] AS g1
-                  │  └ Collect[doc.t1 | [o['i']] | true]
-                  └ Rename[id, o['i']] AS b
-                    └ Collect[doc.t1 | [id, o['i']] | true]
+                  ├ Rename[o] AS g1
+                  │  └ Collect[doc.t1 | [o] | true]
+                  └ Rename[id, o] AS b
+                    └ Collect[doc.t1 | [id, o] | true]
```

Comment:
@mfussenegger Is this change in plan moving from `o[`i`]` to `o` a problem ? How is that executed on runtime? Will this emit larger rows through the BatchIterators?

Reply:
You can ignore that, it's fixed.